server/sonar-web/src/main/webapp/WEB-INF/app/models/snapshot.rb

```diff
@@ -21,13 +21,16 @@ class Snapshot < ActiveRecord::Base
   include Resourceable
 
   belongs_to :project, :class_name => 'Project', :foreign_key => 'component_uuid',:primary_key => 'uuid'
-  belongs_to :root_project, :class_name => 'Project', :foreign_key => 'root_component_uuid',:primary_key => 'uuid'
 
   has_many :events, :class_name => 'Event', :foreign_key => 'analysis_uuid', :primary_key => 'uuid', :dependent => :destroy, :order => 'event_date DESC'
 
   STATUS_UNPROCESSED = 'U'
   STATUS_PROCESSED = 'P'
 
+  def root_project
```

Comment:
I did a full text search, seems root_project is barely used anymore (found 2 locations)